Wikipedia/Code/PreviewWebViewContainer.m

```diff
@@ -58,14 +57,11 @@ - (void)awakeFromNib {
     [super awakeFromNib];
     WKWebView *webview = [[WKWebView alloc] initWithFrame:CGRectZero configuration:[self configuration]];
     webview.translatesAutoresizingMaskIntoConstraints = NO;
-    [self addSubview:webview];
+    [self wmf_addSubviewWithConstraintsToEdges:webview];
```

Comment:
The `wmf_addSubview` way using left and right instead of leading and trailing. Should be just switch it to use leading and trailing too?

Reply:
It doesn't really matter in this case since both edges are zero- it's just potentially confusing to have non-directional insets (`UIEdgeInsets` - `NSDirectionalInsets` is iOS 11 only) and directional layout anchors for the main method. We can try leading/trailing and see if there are any issues since that should be the default over left/right.